### PR TITLE
Ensure broker details are passed through to settings file

### PR DESCRIPTION
**Pull request by knolleary**
This plumbs through the broker url/username/password env vars through to the settings file to ensure the Project Link nodes receive a valid configuration

### CODE REVIEW

**Review by Steve-Mcl — APPROVED** [review]
yep - looks good.
will pull and test as soon as merged

**Comment by knolleary** [comment]
Hold off merging - unit test failure

**Pull request merged**
